proto/buf/alpha/registry/v1alpha1/plugin_curation.proto

```diff
@@ -89,6 +90,25 @@ enum PythonPackageType {
   PYTHON_PACKAGE_TYPE_STUB_ONLY = 2;
 }
 
+// DotnetTargetFramework specifies the target framework for dotnet plugins.
+// Reference: https://learn.microsoft.com/en-us/dotnet/standard/frameworks
```

Comment:
do we want to add a comment as to why we've selected this subset of target frameworks?